src/azure-cli/azure/cli/command_modules/eventgrid/tests/latest/test_eventgrid_commands.py

```diff
@@ -670,6 +670,44 @@ def test_event_subscription_with_storagequeuemessage_ttl(self, resource_group):
 
         self.cmd('az eventgrid event-subscription delete --source-resource-id {source_resource_id} --name {event_subscription_name}')
 
+    @ResourceGroupPreparer(name_prefix='clieventgrid', location='centraluseuap')
+    @StorageAccountPreparer(name_prefix='clieventgrid', location='centraluseuap')
```

Comment:
```suggestion
    @ResourceGroupPreparer(name_prefix='clieventgrid', location='centraluseuap')
```
`@StorageAccountPreparer` is not needed here since you created manually below.